pkgs/ffigen/test/code_generator_tests/expected_bindings/_expected_enumclass_bindings.dart

```diff
@@ -18,4 +18,10 @@ enum Constants {
 
   final int value;
   const Constants(this.value);
+
+  static Constants fromValue(int value) => switch (value) {
+        10 => a,
+        -1 => b,
+        _ => throw ArgumentError("Invalid value for Constants: $value"),
```

Comment:
Maybe change this to `"Unknown value for Constants: $value"`. (If someone adds enum values on the C side, but forgets to regenerate the bindings, the value is valid but the bindings don't know about it.)

Reply:
Done